### ISSUE TITLE
errorPlacement not called properly when in groups

### ISSUE THREAD

**Issue by losewin** [jquery-validation/jquery-validation#2298]
So the problem is, I group some inputs but when the other one has value and the other one does not, the `errorPlacement` is not called when I expect that it should be called.
But when Im not putting it on a group the other input errorPlacement is called.
I wan't the two input be required
But as what I check on the validation object in the error list there is a message for the hidden input but the thing is, it is not triggering the `errorPlacement`
```javascript
rules:{
  'departFrom[airportcod]':'required',
  'departFrom[display]':'required',
}
groups:{
  deaprtFrom:'departFrom[airportcod] departFrom[display]'
},
errorPlacement: function (error, element) {
  console.log(error, element)
}
```

**Comment by Arkni** [comment]
Hi @losewin,

If one of the fields is hidden, it will be ignored by the plugin. See [`ignore`](https://jqueryvalidation.org/validate/#ignore) option for more information.

**Comment by losewin** [comment]
I already set the `ignore` to `[]` but still does not do the work. Also I already proved that the validation is working fine because those hidden fields was successfully validated I confirmed this because it was listed on the `errorList` the thing is, it just not triggering the `errorPlacement` which should be

Thanks for the response by the way :)

**Comment by Arkni** [comment]
Hi @losewin,

I would like to help, but I don't really have enough time to spare for this. Could you please debug the issue and see if you can find the culprit? That will help us fix it easily.

Thanks!
